app/models/incomeSourceDetails/IncomeSourceDetailsResponse.scala

```diff
@@ -60,6 +61,11 @@ case class IncomeSourceDetailsModel(nino: String,
     taxYears
   }
 
+  def orderedTaxYearsInWindows(windowWidth: Int)(implicit dateService: DateServiceInterface): List[List[Int]] = {
```

Comment:
you can directly read from appConfig.api1553MaxYears
no actually pass windowWidth as external, this way data will be always looks the same etc